dashboard/app/models/pd/workshop.rb

```diff
@@ -35,16 +35,39 @@ class Pd::Workshop < ActiveRecord::Base
     COURSE_ECS = 'ECS',
     COURSE_CS_IN_A = 'CSinA',
     COURSE_CS_IN_S = 'CSinS',
-    COURSE_CSD = 'CSD'
+    COURSE_CSD = 'CSD',
+    COURSE_COUNSELOR_ADMIN = 'Counselor/Admin'
   ]
 
   STATE_NOT_STARTED = 'Not Started'
```

Comment:
Might be nicer to define this as

STATES = [
NOT_STARTED,
IN_PROGRESS,
ENDED
] for better readability